source/extensions/tracers/zipkin/util.cc

```diff
@@ -22,6 +23,12 @@ uint64_t Util::generateRandom64(TimeSource& time_source) {
   return rand_64();
 }
 
+ProtobufWkt::Value Util::uint64Value(uint64_t value, Replacements& replacements) {
+  const std::string string_value = std::to_string(value);
+  replacements.push_back({absl::StrCat("\"", string_value, "\""), string_value});
```

Comment:
I'm assuming there is no possibility that an integer that is called for replacement in a replacement context conflicts with an unrelated integer which happens to share the value? i.e. we have a replacement for 42, and accidentally substitute an unrelated 42?

Reply:
I think I'll add a precautionary by adding the field name as well.

Reply:
This actually happened. We need a follow-on PR that adds the field name.

Reply:
See: https://github.com/envoyproxy/envoy/pull/11374